### PR TITLE
update APK path in devicelab tests

### PR DESCRIPTION
**Pull request by yjbanov**
/cc @mravn-google

### CODE REVIEW

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -274,7 +274,12 @@ class CompileTest {
         watch.start();
         await flutter('build', options: options);
         watch.stop();
-        releaseSizeInBytes = await file('$cwd/build/app/outputs/apk/app-release.apk').length();
+        File apk = file('$cwd/build/app/outputs/apk/release/app-release.apk');
```
Comment (mravn-google) [inline dev/devicelab/lib/tasks/perf_tests.dart]: This works. We could also just take the copy at `build/app/outputs/apk/app.apk` like the Flutter tools do?

**Review by mravn-google — APPROVED** [review]
LGTM

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 1 above)
Comment (yjbanov) [reply dev/devicelab/lib/tasks/perf_tests.dart]: Hmm, will check this option too.

**Comment by yjbanov** [comment]
Submitting this with a manual fix for transitive dependencies to unblock the build.

**Pull request merged**